dedibox-network/rpn/how-to/use-rpn-vpn.mdx

```diff
@@ -11,7 +11,7 @@ It is based on [OpenVPN](/tutorials/install-openvpn/) and you can add the RPN VP
 
 The RPN VPN service allows you to:
 
-* Connect from anywhere in your totally secure and private network    
+* Connect from anywhere in your totally secure and Private Network    
```

Comment:
I think here it isn't a product name (only for Elements)

Reply:
```suggestion
* Connect from anywhere in your totally secure and private network    
```